test/IntegrationTests/TicketService/test_create_custom_fields.py

```diff
@@ -0,0 +1,160 @@
+        self.docker_client = docker.from_env()
+
+        # Ensure MySQL container is running
+        mysql_container = self.docker_client.containers.get(self.MYSQL_CONTAINER_NAME)
+        if mysql_container.status != "running":
+            logger.info("Starting MySQL container for tests.")
+            mysql_container.start()
+
+        # Assuming an endpoint `/addcustomer` to create a new customer
+        customer_url = f"{self.BASE_URL}/addcustomer"
+        response = requests.post(customer_url)
+        self.assertEqual(response.status_code, 200, "Failed to create customer")
+        self.valid_customer_guid = response.json().get("customer_guid")
+        logger.info(f"Valid customer_guid initialized: {self.valid_customer_guid}")
+
+    def test_add_valid_custom_field(self):
+        """Test adding a valid custom field."""
+        url = f"{self.BASE_URL}/custom_fields"
+        data = {
+            "customer_guid": self.valid_customer_guid,
+            "field_name": "test_field",
+            "field_type": "VARCHAR(255)",
+            "required": True
+        }
+        logger.info(f"Testing valid custom field addition with data: {data}")
+        response = requests.post(url, json=data)
+        self.assertEqual(response.status_code, 200, "Failed to add custom field")
+        response_data = response.json()
+        self.assertEqual(response_data["field_name"], "test_field")
+        logger.info("Test case for valid custom field addition passed.")
+
+    def test_add_custom_field_missing_required_field(self):
+        """Test adding a custom field with missing required field."""
+        url = f"{self.BASE_URL}/custom_fields"
+        data = {
+            "customer_guid": self.valid_customer_guid,
+            "field_name": "missing_required_field",
+            "field_type": "VARCHAR(255)"
+            # 'required' field is missing
+        }
+        logger.info(f"Testing missing 'required' field with data: {data}")
+        response = requests.post(url, json=data)
+        self.assertEqual(response.status_code, 422, "Missing required field should result in 422")
+        self.assertIn("field required", response.text.lower())
+        logger.info("Test case for missing required field passed.")
+
+    def test_add_custom_field_invalid_customer_guid(self):
+        """Test adding a custom field with invalid customer_guid."""
+        url = f"{self.BASE_URL}/custom_fields"
+        data = {
+            "customer_guid": "256128e2-b963-4019-b870-d6e82db0d631", # Invalid or random customer_guid
+            "field_name": "test_field",
+            "field_type": "VARCHAR(255)",
+            "required": True
+        }
+        logger.info(f"Testing invalid customer GUID with data: {data}")
+        response = requests.post(url, json=data)
+        self.assertEqual(response.status_code, 400, "Invalid customer_guid should result in 400")
+        self.assertIn("Database", response.text)
```

Comment:
CHECK WHETHER CUSSTOMER_GUID SHOULD BE VALID TEXT IS SENT BACK

Reply:
checked and modified sir